src/algoesup/test.py

```diff
@@ -3,6 +3,24 @@
 from typing import Callable
 
 
+def validate_test_table(function: Callable, test_table: list) -> list:
+    """Validate the structure of a test table"""
```

Comment:
I would integrate this code in the `test()` function and avoid the extra complication of error lists.

Reply:
Ok that makes sense.